gfx.cpp

```diff
@@ -5384,9 +5407,9 @@ class GfxInternal
             {
                 D3D12_LIBRARY_DESC library_desc;
                 library->GetDesc(&library_desc);
-                for(uint32_t i = 0; i < library_desc.FunctionCount; i++)
+                for(uint32_t k = 0; k < library_desc.FunctionCount; k++)
```

Comment:
There was already an `i` in the parent for loop, renamed to k to its doesnt clash (theres also a j)